pkg/cmd/pulumi/about/about.go

```diff
@@ -116,6 +117,13 @@ func getSummaryAbout(
 		Errors:        []error{},
 		ErrorMessages: []string{},
 		LogMessage:    formatLogAbout(),
+		Environment:   map[string]string{},
+	}
+	for _, env := range os.Environ() {
```

Comment:
Can we use the same logic that we use `pulumi about env` here?

That will keep this filtered to only the env vars we care about.